.npmrc

```diff
@@ -1 +1,2 @@
-auto-install-peers=false
\ No newline at end of file
+auto-install-peers=false
```

Comment:
we probably also want: https://github.com/ember-cli/ember-cli/blob/master/blueprints/addon/files/.npmrc#L8

can be separate, ofc